haskell/src/Monocle/Backend/Test.hs

```diff
@@ -586,16 +591,16 @@ testTaskDataAdd = withTenant doTest
           td43 = mkTaskData "43"
       void $ I.taskDataAdd [td42, td43]
       -- Ensure only changes 42 and 43 got a Task data associated
-      changes <- I.getChangesByURL (map ("https://fakeprovider/" <>) ["42", "43", "44"])
+      changes <- I.runScanSearch $ I.getChangesByURL (map ("https://fakeprovider/" <>) ["42", "43", "44"])
       assertEqual'
         "Check adding matching taskData"
-        [ ("42", Just [I.toELKTaskData td42]),
+        [ ("44", Nothing),
```

Comment:
@morucci If I understand correctly that is the correct order, e.g. the API returns the most recent first. Though I'm not sure why when we switched to the BH.scanSearch, the result somehow got reversed...

Reply:
Yes, but I wonder why #634 changed the order.